client/my-sites/plans/plan-overview/plan-features/index.jsx

```diff
@@ -2,21 +2,82 @@
  * External dependencies
  */
 import React from 'react';
+import page from 'page';
 
 /**
  * Internal dependencies
  */
-import Card from 'components/card';
+import PlanFeature from './feature'
 import SectionHeader from 'components/section-header';
 
 const PlanFeatures = React.createClass( {
+	propTypes: {
+		selectedSite: React.PropTypes.oneOfType( [
+			React.PropTypes.object,
+			React.PropTypes.bool
+		] ).isRequired
+	},
+
+	goToAddDomain() {
+		page( `/domains/add/${ this.props.selectedSite.slug }` );
+	},
+
+	goToCustomizer() {
+		page( `/customize/${ this.props.selectedSite.slug }` );
+	},
+
+	goToGoogleAnalytics() {
+		page( `/settings/analytics/${ this.props.selectedSite.slug }` );
+	},
+
+	goToPlugins() {
+		page( `/plugins/${ this.props.selectedSite.slug }` );
+	},
+
 	render() {
 		return (
 			<div>
 				<SectionHeader label={ this.translate( "Your Site's Features" ) } />
-				<Card>
-					...
-				</Card>
+
+				<PlanFeature
+					button={ { label: this.translate( 'Find a Domain' ), onClick: this.goToAddDomain } }
```

Comment:
What do you think about passing an `href` property through `button` instead using `onClick` handlers with `page`? This will cause `<Button />` to render with an anchor tag, which has some upside in terms of accessibility.